web-registry/src/pages/CreateCreditClass/CreateCreditClass.tsx

```diff
@@ -23,6 +25,10 @@ export const CreateCreditClass = (): JSX.Element => {
     console.log('handleSubmit', values);
   }
 
+  if (!isCreditClassCreator) {
```

Comment:
if the person is not a credit class creator, we hide the create credit class page. normally, the person would have to click to get to this page. but adding this makes sure that even if they have the URL cached, they wouldn't be able to see anything.